rpcs3/rpcs3qt/debugger_list.cpp

```diff
@@ -99,6 +101,8 @@ void debugger_list::ShowAddress(u32 addr, bool force)
 			if (!vm::check_addr(cpu_offset + pc, 4))
 			{
 				item(i)->setText((IsBreakpoint(pc) ? ">> " : "   ") + qstr(fmt::format("[%08x]  ?? ?? ?? ??:", pc)));
+				item(i)->setForeground(palette().color(foregroundRole()));
```

Comment:
wouldn't it make more sense to simply move the entire section that sets these to the beginning of the loop scope ?

Reply:
Review the latest changes please.

Reply:
when will you stop producing the most unreadable code in history

Reply:
Okay for now.